src/components/Contentful/HomePage/NewHomeLoansByCategoryGrid.vue

```diff
@@ -1,35 +1,21 @@
 <template>
-	<section-with-background-classic
-		:background-content="background"
-		:theme-name="themeName"
-		:vertical-padding="verticalPadding"
-	>
-		<template #content>
-			<kv-page-container>
-				<div>
-					<kiva-multi-category-grid
-						:contentful-loan-channels="contentfulLoanChannels"
-						:loan-display-settings="loanDisplaySettings"
-						:new-home-exp="true"
-					/>
-				</div>
-			</kv-page-container>
-		</template>
-	</section-with-background-classic>
+	<div>
+		<kiva-multi-category-grid
+			:contentful-loan-channels="contentfulLoanChannels"
+			:loan-display-settings="loanDisplaySettings"
+			:new-home-exp="true"
+		/>
+	</div>
 </template>
 
 <script>
 import contentfulStylesMixin from '@/plugins/contentful-ui-setting-styles-mixin';
 import KivaMultiCategoryGrid from '@/components/Homepage/HomeExp/KivaMultiCategoryGrid';
-import SectionWithBackgroundClassic from '@/components/Contentful/SectionWithBackgroundClassic';
-import KvPageContainer from '~/@kiva/kv-components/vue/KvPageContainer';
```

Comment:
SectionWithBackgroundClassic and KvPageContainer are both useful components to use here, as they let the spacing, theme, and background all be controlled from Contentful while keeping the content aligned with the rest of the page. We should try to keep both of these if possible. Were they causing styling issues?

Reply:
I have reverted this change!